test/y2network/config_reader/sysconfig_test.rb

```diff
@@ -109,4 +109,16 @@
       end
     end
   end
+
+  describe "#forward_ipv4?" do
+    it "returns true when IPv4 forwarding is allowed" do
+      expect(reader.send(:forward_ipv4?)).to be_truthy
+    end
+  end
+
+  describe "#forward_ipv6?" do
+    it "returns false when IPv6 forwarding is disabled" do
```

Comment:
You should add a test when the forwarding is enabled.

Reply:
well, I'd need two sysctl files for that ... and i was lazy